app/services/auth/Auth.js

```diff
@@ -0,0 +1,70 @@
+import Auth0Lock from 'auth0-lock';
+import { AUTH_CONFIG } from './auth0-config';
+import history from './history';
+
+export default class Auth {
+  constructor() {
+    this.login = this.login.bind(this);
+    this.logout = this.logout.bind(this);
+    this.handleAuthentication = this.handleAuthentication.bind(this);
+    this.isAuthenticated = this.isAuthenticated.bind(this);
+  }
+
+  lock = new Auth0Lock(AUTH_CONFIG.clientId, AUTH_CONFIG.domain, {
```

Comment:
I'd probably init this in the constructor. I'm not sure if you instantiated multiple `Auth` classes if they would all share this instance of `Auth0Lock`.